### ISSUE TITLE
Need API to show activity

### ISSUE THREAD

**Issue by dlabrecq** [openshiftio/openshift.io#1631]
I'm in the process of implementing the UXD design below for the my spaces page.
https://redhat.invisionapp.com/share/DQDSL22JA#/screens/261389228

There is no API we can use to show activity. Unfortunately, we have to omit this feature from the new my spaces page.

**Comment by aslakknutsen** [comment]
@dlabrecq I'll assume this is about the 'sound wave' image in the list. Got any details of what that is supposed to represent?

**Comment by dlabrecq** [comment]
The sound wave image is basically an activity indicator. The more activity, the larger sound wave shown.

I'll defer to @bdellasc here, but we've been wanting to display user activity for some time.

**Comment by aslakknutsen** [comment]
@dlabrecq yeah. I'm looking for what do we constitute as 'activity'? Whats the scale of the wave, number per week, per month?
